### DISCUSSION TITLE
GraphCMS GraphQL Queries not returning "parsed"

### DISCUSSION THREAD

**Discussion by fjshomeplate** [gatsbyjs/gatsby#28626] [Help] [answered]
I am using GraphCMS for my website, and the queries aren't coming back "parsed." {data.cms.nodes.productName} returns nothing. Here is my code: 

import React from "react"
import { useStaticQuery, graphql } from "gatsby"

const productThumbnail= () => {
  const data = useStaticQuery(graphql`
    {
      allGraphCmsProducts {
        nodes {
          productName
        }
      }
    }
  `)
  return <pre>{data.allGraphCmsProducts.nodes.productName}</pre>
}
export default productThumbnail

**Comment by vladar** [comment, marked as answer]
I think you should ask this question in GraphCMS source plugin repo: https://github.com/GraphCMS/gatsby-source-graphcms/issues

But also why is it `data.cms.nodes.productName`? When it should be `data.allGraphCmsProducts.nodes.productName` given your query?

**Reply by fjshomeplate** [reply]
Thank you for catching that!
